applications/rubintv-dev/values-summit.yaml

```diff
@@ -0,0 +1,40 @@
+rubintv:
+  siteTag: "summit"
+
+  imagePullSecrets:
+  - name: pull-secret
+
+  frontend:
+    debug: true
+    env:
+    - name: S3_ENDPOINT_URL
+      value: "https://s3.rubintv.cp.lsst.org"
```

Comment:
Tip: You can use some YAML tricks to not duplicate information that's used elsewhere in the file. See: https://github.com/lsst-sqre/phalanx/blob/main/applications/simonyitel/values-tucson-teststand.yaml at the `ccoods` and `mtaos` entries.

Reply:
Nice!! I didn't know about this trick, thank you one more time Michael :pray:, I'll make the change.